### DISCUSSION TITLE
asp-dotnet-core-in-action-version-3-is-now-available/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#43] [Announcements]
# asp-dotnet-core-in-action-version-3-is-now-available/

ASP.NET Core in Action, version 3, is available now from Manning's Early Access Program, and is fully updated to .NET 7.

https://andrewlock.net/asp-dotnet-core-in-action-version-3-is-now-available/

**Comment by mickdelaney** [comment]
Congrats Andrew !!! ;-)

**Reply by andrewlock** [reply]
Thanks! :smile:

**Comment by ricardodemauro** [comment]
Congrats!
